src/com/sony/simdetect/SimDetectService.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019-2020 The LineageOS Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Rewritten in Kotlin by Pavel Dubrova <pashadubrova@gmail.com>
+ */
+
+package com.sony.simdetect
+
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.os.Handler
+import android.os.IBinder
+import android.os.Looper
+import android.os.Message
+import android.os.UEventObserver
+
+import com.android.internal.R
+import com.android.internal.telephony.uicc.UiccSlot
+
+class SimDetectService : Service() {
+    private val mLock: Any = Any()
```

Comment:
I think 'm' prefix shouldn't be here anymore?
![image](https://user-images.githubusercontent.com/6830979/72397747-05ec5e00-3741-11ea-8d23-0df49b002242.png)

Reply:
Also do you really need to do ": Any" if you're doing Any() right after it?

Reply:
same for mSimDetectEventObserver, you should be able to drop "object : ".